README.md

```diff
@@ -1 +1,134 @@
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Mobile Porfolio <a name="about-project"></a>
+
+**Mobile Portfolio** is a place to show my web development projects.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+HTML, CSS
+
+### Key Features <a name="key-features"></a>
+
+- **Header**
+- **Work Section**
+- **About me**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name=""></a>
+
+- [Live Demo Link](https://moses-chibuike.github.io/Football-Conference/index.html)
+- Right click > Inspect > Choose Mobile version
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+You can check the repo and see the desgin of the page
+
+### Prerequisites
+
+Setup usual dev environment: VSCode, Live Server
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Moses-chibuike/Portfolio-microverse.git
+```
+
+### Usage
+
+To run the project, Run `Live Server` from VSCode.
+
+### Run tests
+
+Coming..
+
+
+### Deployment
+
+You can deploy this project using: Netlify, and some other hosting service: <a href="https://glitch.com/">glitch</a>, <a href="https://replit.com/">replit</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Aya**
+
+- GitHub: [@Moses-chibuike](https://github.com/Moses-chibuike/)
+- Twitter: [Ezechukwuchibu2](https://twitter.com/Ezechukwuchibu2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ezechukwu-chibuike/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Contact me section
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and my coding partners
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] While you did a good job adding the required information on your README :100: I think it would be better if you give the appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the Acknowledgments section.